next-tavla/app/(board)/[id]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import { getBoard, getOrganizationLogoWithBoard } from 'Admin/utils/firebase'
+import { notFound } from 'next/navigation'
+import { TBoard } from 'types/settings'
+import { upgradeBoard } from 'utils/converters'
+import classes from 'styles/pages/board.module.css'
+import { Header } from 'components/Header'
+import { Board } from 'Board/scenarios/Board'
+import { Footer } from 'components/Footer'
+import { Pinger } from 'components/Pinger'
+
+async function BoardPage({ params }: { params: { id: string } }) {
+    const { id } = params
+
+    let board: TBoard | undefined = await getBoard(id)
+
+    if (!board) return notFound()
```

Comment:
Vi har ikke noe `notFound` side som håndterer dette?

https://nextjs.org/docs/app/api-reference/file-conventions/not-found